<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crop an image</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonInputFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input image file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInputFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>921</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCrop">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>961</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>431</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutputFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output image file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>921</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_range">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_zto">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>to:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_z">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Full?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_x1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_z1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_x2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_y1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_y">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Full?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_xrange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X range: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_zrange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z range: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_yrange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y range: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_xto">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>to:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_yfrom">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>from:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_xfrom">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>from:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_z2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_x">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Full?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_yto">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>to:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_zfrom">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>from:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_y2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_blockchoice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_range">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Ranges</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_centre">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Centre and size</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_centre">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_centre">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Centre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_size">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_y">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_z">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_xcentre">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ycentre">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_zcentre">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_xsize">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ysize">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_zsize">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_shape">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="radioButton_block">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Block</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_shape</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_sphere">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Sphere</string>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_shape</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_sphere">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_z_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_centre_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Centre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_y_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_x_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_x0">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_y0">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_z0">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_R">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_radius">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1017</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOutputFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outputFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonInputFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>inputFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCrop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cropper()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_y</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>checkBox_y()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_z</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>checkBox_z()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_x</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>checkBox_x()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>outputFileSelecter()</slot>
  <slot>inputFileSelecter()</slot>
  <slot>cropper()</slot>
  <slot>checkBox_x()</slot>
  <slot>checkBox_y()</slot>
  <slot>checkBox_z()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup_shape"/>
 </buttongroups>
</ui>
